praw/reddit.py

```diff
@@ -567,11 +568,19 @@ def _objectify_request(
         :param params: The query parameters to add to the request (default:
             None).
         :param path: The path to fetch.
+        :param json: JSON-serializable object to send in the body
+            of the request with a Content-Type header of application/json
+            (default: None). If ``json`` is provided, ``data`` should not be.
 
         """
         return self._objector.objectify(
             self.request(
-                data=data, files=files, method=method, params=params, path=path
+                data=data,
+                files=files,
+                method=method,
+                params=params,
+                path=path,
+                json=json,
```

Comment:
This should be sorted as well.

```suggestion
                json=json,
                method=method,
                params=params,
                path=path,
```